conda_build_prepare/conda_cmds.py

```diff
@@ -150,11 +150,31 @@ def _add_extra_tags_if_exist(package_dir, repo_path):
                     continue
         print()
 
+# A wrapper for subprocess.check_output printing stdout in case of process
+# error. By default it isn't printed and can make debugging much harder.
+def _check_output(command, **kwargs):
+    try:
+        return subprocess.check_output(command.split(), encoding='utf-8',
+                **kwargs).strip()
```

Comment:
Do we want sth like `stderr=subprocess.STDOUT`?

Reply:
Definitely no. If we do anything with *stdout* (like capturing rendered recipe) then returning it mixed with *stderr* isn't an option.